docs/quickstart.md

```diff
@@ -15,6 +15,16 @@ with the only help of an Elemental Teal iso
 
 ---
 
+### What is the Rancher Elemental Stack ?
+
+The Elemental Stack consists of some packages on top of SLE Micro for Rancher
+
+- **elemental-toolkit** - includes a set of OS utilities to enable OS management via containers. Includes dracut modules, bootloader configuration, cloud-init style configuration services, etc.
+- **elemental-operator** - this connects to Rancher Manager and handles machineRegistration and machineInventory CRDs
```

Comment:
elemental-operator is not "on top of SLE Micro for Rancher" and should be listed separately

Reply:
I agree, that is not related to SLE Micro, but it would great anyway have listed it here on this `quickstart` page

Reply:
I gave a little bit more context instead of "on top of SLE Micro". I'm not in love with what I wrote but it's early on a monday.